packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -309,29 +309,30 @@ export class DefaultWeb3Handler
         txObject
       )}], defaultBlock: [${defaultBlock}]`
     )
-    // First generate the internalTx calldata
-    const internalCalldata = this.getTransactionCalldata(
-      this.getTimestamp(),
-      0,
-      txObject['to'],
-      txObject['data'],
-      txObject['from'],
-      ZERO_ADDRESS,
-      true
-    )
-
-    log.debug(internalCalldata)
-    // Then estimate the gas
-    const response = await this.context.provider.send(
-      Web3RpcMethods.estimateGas,
-      [
-        {
-          from: this.context.wallet.address,
-          to: this.context.executionManager.address,
-          data: internalCalldata,
-        },
-      ]
-    )
+    // // First generate the internalTx calldata
+    // const internalCalldata = this.getTransactionCalldata(
+    //   this.getTimestamp(),
+    //   0,
+    //   txObject['to'],
+    //   txObject['data'],
+    //   txObject['from'],
+    //   ZERO_ADDRESS,
+    //   true
+    // )
+    //
+    // log.debug(internalCalldata)
+    // // Then estimate the gas
+    // const response = await this.context.provider.send(
+    //   Web3RpcMethods.estimateGas,
+    //   [
+    //     {
+    //       from: this.context.wallet.address,
+    //       to: this.context.executionManager.address,
+    //       data: internalCalldata,
+    //     },
+    //   ]
+    // )
```

Comment:
reminder that this is commented out if it was unintentional--fine by me to leave though

Reply:
👍 leaving it in because we may want to uncomment in a bit.